package.json

```diff
@@ -49,6 +49,11 @@
     "@emotion/css": "^11.11.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@fortawesome/fontawesome-svg-core": "^6.6.0",
+    "@fortawesome/free-brands-svg-icons": "^6.6.0",
```

Comment:
We are not using these packages (`@fortawesome/free-solid-svg-icons` and `@fortawesome/free-regular-svg-icons`). I think they should be removed.